pyiron_base/job/core.py

```diff
@@ -739,18 +739,34 @@ def copy_to(self, project, new_job_name=None, new_database_entry=True, copy_file
 
         Args:
             project (JobCore/ProjectHDFio/Project): project to copy the job to
-            new_database_entry (bool): [True/False] to create a new database entry - default True
+            input_only (bool): [True/False] Whether to copy only the input. (Default is False.)
+            new_database_entry (bool): [True/False] Whether to create a new database entry. If input_only is True then
+                new_database_entry is False. (Default is True.)
             copy_files (bool): [True/False] copy the files inside the working directory - default True
 
         Returns:
             JobCore: JobCore object pointing to the new location.
         """
-        new_job_core, _, _, _ = self._internal_copy_to(
+        # Update flags
+        if input_only and new_database_entry:
+            new_database_entry = False
+
+        new_job_core, _, _, reload = self._internal_copy_to(
             project=project,
             new_job_name=new_job_name,
             new_database_entry=new_database_entry,
             copy_files=copy_files
         )
+        if reload_flag:
+            return new_job_core
+
+        # Remove output if it should not be copied
+        if input_only:
+            if "output" in new_job_core.project_hdf5.list_groups():
```

Comment:
*sigh* we really, really, really need to get v1.0 out and fix our janky storage arrangement.

So since we never updated all the code to stop using `GenericInput` and `GenericOutput`, when I make new classes that use `DataContainer` for IO, I have to give them paths *other* than `"input"` and `"output"`. 

I guess here we can make a simple fix like

```python
for group in new_job_core.project_hdf5.list_groups():
    if "output" in groups:
        blah blah
```

That will cover us for now, but I am really longing for a more consistent data management scheme!

Reply:
I don't quite understand this.  This is catch all groups in HDF that have "output" somewhere in the name?

Reply:
Yeah, that's my concern; that there are "output_foo" fields that should be excluded along with "output".

Reply:
Ah, that is new functionality then, but I can add it, sure.